pybind/wallet3/wallet.cpp

```diff
@@ -0,0 +1,28 @@
+#include "../common.hpp"
+
+#include <wallet3/wallet.hpp>
+#include <wallet3/default_daemon_comms.hpp>
+#include <wallet3/keyring.hpp>
+#include <wallet3/config/config.hpp>
+#include <oxenmq/oxenmq.h>
+
+namespace wallet3
+{
+  void
+  Wallet_Init(py::module& mod)
+  {
+    py::class_<wallet::Wallet, std::shared_ptr<wallet::Wallet>>(mod, "Wallet")
+      .def(py::init([](std::string& wallet_name, wallet::Keyring& kr, wallet::Config& config) {
+        auto keyring = std::make_shared<wallet::Keyring>(std::move(kr));
+        auto& comms_config = config.daemon;
+        auto& omq_rpc_config = config.omq_rpc;
+        auto oxenmq = std::make_shared<oxenmq::OxenMQ>();
+        auto comms = std::make_shared<wallet::DefaultDaemonComms>(oxenmq, comms_config);
+        return wallet::Wallet::create(oxenmq, std::move(keyring), nullptr, comms, wallet_name + ".sqlite", "", std::move(config));
```

Comment:
This `move` on config with the lvalue reference argument seems undesirable.  I'm not sure what `Wallet::create` takes, but if it wants a copy we should take the `config` argument to the function here by value instead of reference (and if it just wants a const lvalue reference, then we should take by const lvalue reference and not move).

Reply:
The wallet contains a config member, which calls a copy constructor on config during the constructor of wallet. So would passing a const reference to this pybind init mean we only get the single copy which is done by wallets constructor. Can we talk about this in a call?